docs/source-2.0/spec/json-ast.rst

```diff
@@ -496,7 +496,7 @@ shapes defined in JSON support the same properties as the Smithy IDL.
       - Map<String, :ref:`AST shape reference <ast-shape-reference>`>
       - Defines identifier names and shape IDs used to identify the resource.
     * - :ref:`properties <resource-properties>`
-      - :ref:`AST shape reference <ast-shape-reference>`
+      - Map<String, :ref:`shape ID <shape-id>`>
```

Comment:
This should be a map of AST shape references instead like `identifiers`, [as can be seen here](https://github.com/awslabs/smithy/blob/c8e8ae3c4e667feecc4205a21f3773c26943b70d/smithy-model/src/test/resources/software/amazon/smithy/model/loader/valid/elided-members.json#L183).

```
Map<String, :ref:`AST shape reference <ast-shape-reference>`>
```

Reply:
Thanks @kstich, seeing that example helps, since the use of ref-to-id indirection is not present in the IDL (its an AST-specific concept). I'll update the PR accordingly :)